endOfEpoch/shardchain/trigger.go

```diff
@@ -177,6 +179,10 @@ func (t *trigger) ReceivedHeader(header data.HeaderHandler) {
 		return
 	}
 
+	if metaHdr.Epoch == t.epoch && len(metaHdr.EndOfEpoch.LastFinalizedHeaders) > 0 {
```

Comment:
can you extract the condition in named variable and use that instead?

Reply:
done.